homework_4/event_settler/main.go

```diff
@@ -0,0 +1,72 @@
+package main
+
+import (
+	"bytes"
+	"encoding/json"
+	"github.com/jelenanemcic/code-cadets-2021/homework_4/event_settler/models"
+	"github.com/pkg/errors"
+	"io"
+	"log"
+	"math/rand"
+	"net/http"
+)
+
+func main() {
+	var settled []string
+
+	response, err := http.Get("http://127.0.0.1:8081/bets?status=active")
```

Comment:
I believe this is okay within the scope of this homework but URL here (and one on L54) could be passed in as program arguments instead of being hardcoded.